### ISSUE TITLE
add ENV variable to odh notebook controller REQUESTS_CA_BUNDLE to point to cluster CA bundle

### ISSUE THREAD

**Issue by shalberd** [opendatahub-io/kubeflow#105]
/kind feature

**Why you need this feature:**
Currently, ODH notebook controller looks at the cluster proxy config and auto-injects ENV variables related to proxy and trusted CA / self-signed server certs handling into spawned notebooks. PIP_CERT is used to reference the mix-in bundle of system-level publicly-trusted and additonal trusted CAs (including, depending on the cluster and environment: self-signed certificates) for https connections.

https://github.com/opendatahub-io/kubeflow/blob/v1.6-branch/components/odh-notebook-controller/controllers/notebook_webhook.go#L282

@Jooho I can confirm that the auto-injection of the envs and also the bundle CA file into the container is working very nicely, [great job](https://github.com/opendatahub-io/kubeflow/pull/43/files).
Python requests.get is often used in python data science to download files from servers via https. When the endpoints live on servers with either self-signed certificates or with private PKI based certificates, it's good practice to make python use env var
REQUESTS_CA_BUNDLE to refer to the full path of the CA bundle file for SSL trust.

https://requests.readthedocs.io/en/latest/user/advanced/

Currently, without that ENV variable, the error with requests.get from any notebook e.g.

```
import requests

x = requests.get('https://myprivatepkibasedhost.com/test/download.xls')
print(x.status_code)
```

is as follows:

```
Error : SSLError - HTTPSConnectionPool(host='myprivatepkibasedhost.com', port=443): Max retries 
exceeded with url: /test/download.xls (Caused by SSLError(SSLCertVerificationError (1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1129)')))
```

**Describe the solution you'd like:**
Also add env var REQUESTS_CA_BUNDLE to the notebooks via notebook webhook
i.e.
`proxyEnvVars["REQUESTS_CA_BUNDLE"] = "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"`

**Anything else you would like to add:**
https://requests.readthedocs.io/en/latest/user/advanced/#ssl-cert-verification

manually setting that env var works

```
import requests
import os
os.environ["REQUESTS_CA_BUNDLE"] = "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"

x = requests.get('https://myprivatepkibasedhost.com/test/download.xls')
print(x.status_code)
```

output / response status code:
`200`

**Comment by shalberd** [comment]
added a ready PR, not too much involved

**Comment by shalberd** [comment]
implemented in Kubeflow v1.6 branch by https://github.com/opendatahub-io/kubeflow/pull/108
